docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -875,6 +887,76 @@ To process the received search command on the server side, follow these steps:
 > This feature supports Tizen 5.0 and Higher for Mobile and Wearable.
 
 
+## Updating and Retrieving Abilities
+To update the abilities on the server side:
+
+1. Create the media controller server handle using the `mc_server_create()`:
```

Comment:
1. Create the media controller server handle using `mc_server_create()`:

Reply:
Done